projects/core/src/base-core.module.ts

```diff
@@ -41,6 +42,7 @@ import { HttpErrorHandlerModule } from './error-handling';
 
     /* This module should be imported by default starting from version 7.0 (CXSPA-3680)*/
     //ErrorHandlingModule.forRoot(),
+    EffectsErrorHandlerModule.forRoot(),
```

Comment:
Just an idea. no need to change.
IMHO it would be good not to "pollute" this BaseCoreModule with fine-grained modules, but rather group them. To be discussed.

Reply:
I think it's a good idea to revisit the base core module once we're done with the error-handling feature. This way, we can get a better understanding of the whole context and also explore the possibility of other modules that could be added in the meantime.